sdcm/nemesis.py

```diff
@@ -487,6 +487,8 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         the authenticator will be reset back in the end of nemesis.
         """
         self._set_current_disruption('SwitchBetweenPasswordAuthAndSaslauthdAuth %s' % self.target_node)
+        if not self.cluster.nodes[0].is_enterprise:
```

Comment:
better to check this after the block `if not self.cluster.params.get('prepare_saslauthd'):`
since in order to check whether `is_enterprise` is true or false we will need to ssh to the node.
This is unnecessary if `prepare_saslauthd` is not set to true.

Reply:
Fixed.